wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/PluginFQNParser.java

```diff
@@ -37,6 +41,12 @@
 @Beta
 public class PluginFQNParser {
 
+  private static final String INCORRECT_PLUGIN_FORMAT_TEMPLATE =
+      "Plugin '%s' has incorrect format. Should be: 'registryURL/publisher/name/version' or 'registryURL/name/version'";
```

Comment:
I think you mean `publisher/name/version` for the second example here.

Reply:
yes, changed